backend/src/server.ts

```
@@ -0,0 +1,206 @@
+    environment: config.server.nodeEnv,
+  });
+});
+
+// 404 handler
+app.use(notFoundHandler);
+
+// Error handling middleware (must be last)
+app.use(errorHandler);
+
+// Declare server variable
+let server: any;
```

ESLint

[error] 'server' is never reassigned. Use 'const' instead.

(prefer-const)